lib/fluent/plugin/out_influxdb.rb

```diff
@@ -20,9 +21,10 @@ def initialize
 
   def configure(conf)
     super
-    @influxdb = InfluxDB::Client.new @dbname, host: @host, 
-                                              port: @port, 
-                                              username: @user, 
+    @remove_tag_prefix = conf['remove_tag_prefix']
```

Comment:
Yeah, it won't work in my case. I removed the accidental whitespace below. If this isn't ok, I'll just use my branch.